nextjs-task-system/app/components/dashboard/tasks/task-table.tsx

```
@@ -0,0 +1,285 @@
+      filters.status === "all" ||
+      !filters.status ||
+      task.status === filters.status;
+    const matchesPriority =
+      filters.priority === "all" ||
+      !filters.priority ||
+      task.priority === filters.priority;
+    const matchesSearch =
+      !filters.search ||
+      task.title.toLowerCase().includes(filters.search.toLowerCase()) ||
+      (task.description &&
+        task.description.toLowerCase().includes(filters.search.toLowerCase()));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)